R/04-normalize.R

```diff
@@ -120,28 +120,28 @@ gimap_normalize <- function(.data = NULL,
     dplyr::filter(norm_ctrl_flag == "negative_control") %>%
     dplyr::select(pg_ids, dplyr::starts_with("late"))
 
-  # TODO: There's one main median that's found by taking the median of the medians?
-  neg_control_median <- median(apply(neg_control_median_df[, -1], 1, median))
+  # TODO: They find a median for each rep, so apply across columns
+  neg_control_median <- apply(neg_control_median_df[, -1], 2, median)
 
   # First and second adjustments to LFC
-  lfc_df <- lfc_df %>%
+  lfc_df_withAdjs <- late_vs_plasmid_df %>%
```

Comment:
Such a minor minor comment I'm going to make here but can we do `lfc_df_adj` so it sticks with the style?

Reply:
Yeah I can change that!